cache/demux.go

```diff
@@ -151,11 +152,26 @@ func (d *demux) Broadcast(events []*clientv3.Event) {
 	}
 }
 
+func (d *demux) SetProgressRev(rev int64) {
```

Comment:
Instead of dedicated function for setting rev, please modify `Broadcast` to accept `watch response.